### PR TITLE
Update State limits

### PR DESCRIPTION
**Pull request by moritzraho**
## Description
Update the limits sections, introduce usage limits (container level) != quotas (org, not efforced)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by pru55e11 — CHANGES_REQUESTED** [review]
Please update the Feature Matrix to include all the limits for state, including the format limits for keys and values.

**Comment by moritzraho** [comment]
@pru55e11 done

File: src/pages/guides/application_state.md [hunk 1]
```diff
@@ -141,39 +141,42 @@ The default region is `amer`, to access another region, you can use the `--regio
...
-Quotas are limits that depend on the organization's entitlements. Every organization with App Builder access is entitled to at least 1 State quota.
+Every organization with App Builder access is entitled to at least 1 State quota.
 
 At the organization level, 1 quota provides:
 
 - 200GB/month bandwidth usage (~5MB/min): `bandwidth usage = bytes uploaded + bytes downloaded`
 - 1GB storage: `storage usage = 2 * key_sizes + value_sizes`
 
-The quota is shared for all State containers in the organization, across all regions. It is not enforced for now, just tracked.
+The quota is shared for all State containers in the organization, across all regions. It is not enforced, just tracked for billing purpose.
```
Comment (shazron) [inline src/pages/guides/application_state.md:176]: do we want to tell people it's not enforced? 

Maybe:
`The quota is shared for all State containers in the organization, across all regions and is tracked for billing purposes`

Comment (moritzraho) [reply src/pages/guides/application_state.md]: thanks done

**Review by shazron — APPROVED** [review]

**Review by pru55e11 — APPROVED** [review]
Thanks for updating! LGTM

**Pull request merged**